Utilities/MCStepLogger/include/MCStepLogger/AnalysisFile.h

```diff
@@ -0,0 +1,95 @@
+/* Interface to files produced with the analysis framework
+ * Write files, read them form disk and provide access to histograms and meta information
+ */
+
+#ifndef ANALYSISFILE_H_
+#define ANALYSISFILE_H_
+
+#include <string>
+#include <vector>
+
+#include "TH1.h"
+
+namespace o2
+{
+namespace mcstepanalysis
+{
+
+struct AnalysisMetaInfo {
+  std::string mName;
+  std::string mFramework;  // the framework the MCStepLogger file was produced with
+  std::string mLabel;      // label, also shown in plots, especially comparison plots
+  std::string mVersionTag; // to trace the development of the analysis
+
+  ClassDefNV(AnalysisMetaInfo, 1);
+};
+
+class AnalysisMetaInfo;
+class AnalysisMeasure;
+
+class AnalysisFile
```

Comment:
what is this encapsulate concretely? It does not seem to be a fil.

Reply:
Is now an MCAnalysisFileWrapper